lib/dpl/provider/modulus.rb

```diff
@@ -16,7 +16,8 @@ def needs_key?
       end
 
       def push_app
-        context.shell "MODULUS_TOKEN=#{option(:api_key)} modulus deploy -p #{option(:project_name)}"
+        ENV['MODULUS_TOKEN']="#{option(:api_key)}"
```

Comment:
@BanzaiMan Yeah that works too and its cleaner. I redid the PR as #363 and closed this one